src/Services/Catalog/CatalogService.WebApi/Controllers/BookController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using CatalogService.Application.UseCases;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CatalogService.WebApi.Controllers
+{
+    [ApiController]
+    [Route("book")]
+    public class BookController(
+        IMediator mediator) : ControllerBase
+    {
+        private readonly IMediator _mediator = mediator;
+
+        [HttpPost]
+        public async Task<IActionResult> Create([FromBody] CreateBookCommand request, CancellationToken token = default)
+        {
+            var id = await _mediator.Send(request, token);
```

Comment:
## :warning: Required Changes

your book has a image url, how do you thing frontend should save image and return to you a url? it is you who should get real file from frontend, save in your file system or in cloud and save the url

fix it